packages/iov-ethereum/src/serialization.ts

```diff
@@ -26,7 +28,8 @@ import { BlknumForkState, Eip155ChainId, eip155V, toRlp } from "./encoding";
 import { Erc20Options } from "./erc20";
 import { encodeQuantity, encodeQuantityString, fromBcpChainId, normalizeHex } from "./utils";
 
-const { fromHex } = Encoding;
+const { fromHex, toUtf8 } = Encoding;
+const ZERO_AMOUNT = "0";
```

Comment:
In BCP, the term "amount" always includes the unit (TokenTicker). What about `ZERO_ETH_QUANTITY` (BCP naming) or `ZERO_ETH_VALUE` (Ethereum naming)?